### ISSUE TITLE
How to set up optimizer and lr scheduler on diffirent networks and datasets?

### ISSUE THREAD

**Issue by JDKYellowPerson** [KAIR-BAIR/nerfacc#228]
In your code, different networks use different optimizer parameters and lr scheduler, how do I set these parameters on my own network and dataset? Please guide me some tips on setting parameters, thanks!

**Comment by JDKYellowPerson** [comment]
On my own network and dataset, I got this loss loss drop curve and the optimizer and scheduler (same as ```train_mlp_nerf.py```) are:
![2023-05-31 14-58-28 的屏幕截图](https://github.com/KAIR-BAIR/nerfacc/assets/118829471/cd30d84c-5d1d-4eb8-81d5-94590fd432ab)
```python
optimizer = torch.optim.Adam(radiance_field.parameters(), lr=1e-4)
scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer,
                                                 milestones=[
                                                     args.max_steps // 2,
                                                     args.max_steps * 3 // 4,
                                                     args.max_steps * 5 // 6,
                                                     args.max_steps * 9 // 10,
                                                 ],
                                                 gamma=0.33,
                                                 )
```
When lr drops at ```args.max_steps // 2``` and ```args.max_steps * 3 // 4``` , the train_loss declines significantly, and i think it is abnormal. The ```train_mlp_nerf.py``` loss curve is more smooth as below:
![2023-05-31 14-58-05 的屏幕截图](https://github.com/KAIR-BAIR/nerfacc/assets/118829471/51b5f4e1-6d8e-4a53-a4e3-457c7150a1b2)
